### DISCUSSION TITLE
Can Notification sound will be ..text read out . which we will put on text . it read out the text  as  notification

### DISCUSSION THREAD

**Discussion by jkmqtt** [dalchandra/Chegg_Skip_Extension#11] [Ideas]
else if(i==1)
	{
		i=0;
		var sound = new Audio;
		sound.src = "https://assets.mixkit.co/sfx/download/mixkit-arcade-retro-game-over-213.wav";
		sound.play();
	}

**Comment by dalchandra** [comment]
Very good Idea. It also improve to read the question heading.
Instead of 
`var sound = new Audio;`
`sound.src = "https://assets.mixkit.co/sfx/download/mixkit-arcade-retro-game-over-213.wav";`
`sound.play();`
You put 
`var msg = new SpeechSynthesisUtterance('Hello World');`
`window.speechSynthesis.speak(msg);`

You can follow [this](https://stackoverflow.com/questions/15653145/using-google-text-to-speech-in-javascript)

**Reply by jkmqtt** [reply]
this extension works excellent while thei chegg q&a tab is opened.

but when i we are on another tab 
notification sound isn't working. 

this is my chrome settings issue ? 
i am not getting notification sound ..
while tab is opened and minimise.
doing other stuff on laptop .

it will be very interesting to work this extension like ..

**we had opened two - tab 
when questions came on particular tab 
it give notification sound.
like (questions available on math ),
( questions available on physics)**

thank you so much for the reply ☺️

**Reply by dalchandra** [reply]
Try yourself, it is an easy task.

**Reply by jkmqtt** [reply]
i am not from programming background
